src/averageAges.js

```diff
@@ -20,8 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
-}
 
+  const menDiedInThisCentury = people.filter(person => {
+    return century
+      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
+      : person.sex === 'm';
+  });
+
+  const sumMenAges = menDiedInThisCentury
+    .map(manAge => manAge.died - manAge.born)
+    .reduce((sumOfAges, currentAge) => sumOfAges + currentAge, 0);
```

Comment:
you can combine `.map` and `.reduce` together to not iterate several times through the array

```suggestion
  const sumMenAges = menDiedInThisCentury
    .reduce((sumOfAges, person) => sumOfAges + (person.died - person.born), 0);
```